packages/storybook/src/nimble/table-column/menu-button/table-column-menu-button.mdx

```diff
@@ -0,0 +1,34 @@
+import { Controls, Canvas, Meta, Title } from '@storybook/blocks';
+import * as menuButtonColumnStories from './table-column-menu-button.stories';
+import ComponentApisLink from '../../../docs/component-apis-link.mdx';
+import TableColumnApisLink from '../../../docs/table-column-apis-link.mdx';
+import { tableColumnMenuButtonTag } from '../../../../../nimble-components/src/table-column/menu-button';
+
+<Meta of={menuButtonColumnStories} />
+<Title of={menuButtonColumnStories} />
+
+The <Tag name={tableColumnMenuButtonTag}/> renders string values as the content
+of a menu button. A client can specify a slot name where they have provided the
+menu that is associated with the button.
+
+The column is neither sortable nor groupable.
+
+<Canvas of={menuButtonColumnStories.menuButtonColumn} />
+
+## API
+
+<TableColumnApisLink />
+<Controls of={menuButtonColumnStories.menuButtonColumn} />
+<ComponentApisLink />
+
+## Usage
```

Comment:
A couple other ideas for docs:
1. at UIWG Fred mentioned this will often be used like a select column. Should we have 1st class docs for this use case? That could mean:
   - updating the example to show a check icon on the currently selected item
   - describing the use case at a high level in the overview
   - describing how to configure the menu (e.g. which check icon to use, which menu item slot to put it in)
2. usage guidance for when to use this column and when to use the action menu

I think both of these are nice-to-have but not required. It's ultimately @fredvisser's call whether they're needed but feel free to do this in a follow up PR if you're unsure of the value and don't want to block this one.

Reply:
I'll follow up with @fredvisser on what, if any, additional docs we want to add or changes we want to make to the example. For now, I'm resolving the conversation to unblock the PR.